packages/library/src/commonMain/kotlin/io/realm/MutableRealm.kt

```diff
@@ -134,9 +142,27 @@ class MutableRealm : BaseRealm {
      * @param instance The object to create a copy from.
      * @return The managed version of the `instance`.
      */
+    // FIXME Due to lack of throwing C-API this will not throw on duplicate keys, so this
+    //  currently effectively mimics the copyOrUpdate API from realm-java.
+    //  https://github.com/realm/realm-kotlin/issues/192
     fun <T : RealmObject> copyToRealm(instance: T): T {
         return io.realm.internal.copyToRealm(configuration.mediator, realmReference, instance)
     }
+
+    /**
+     * Returns the results of querying for all objects of a specific type.
+     *
+     * The result is live and are thus also reflecting any update to the MutableRealm.
```

Comment:
```suggestion
     * The result is live and thus also reflects any update to the [MutableRealm].
```
Passive -> active voice fix + adding a link.